Example/TOViewController.m

```diff
@@ -112,6 +112,7 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 #endif
     
     TOWebViewController *webViewController = [[TOWebViewController alloc] initWithURL:url];
+    [webViewController setDoneButtonTintColor:[UIColor redColor]];
```

Comment:
Probably best to comment it out by default, so people can see the view controller in its default state initially. :)